mds_loadtest/scripts/loadTestConstants.EXAMPLE.py

```diff
@@ -0,0 +1,2 @@
+BEARER_TOKEN=""
```

Comment:
If this file is trying to simulate what a .env would do, we should probably just use a .env for this.

We have something similar on how to import .env files in python under the backend app (see lines 3-7)
https://github.com/bcgov/mds/blob/develop/python-backend/app/config.py#L7